README.md

```
@@ -2,313 +2,1411 @@
+- [Muhammad Murtaza](https://murtazasarwar.com)
+- [Muhammad Mustafiz Rahman](https://mustafizrahman.vercel.app)[Frontend Developer | Mern Stack Developer]
+- [Muhammad Naeem Tahir](https://muhammadnaeemtahir.github.io)
+- [Muhammad Rashid](https://iamrashy.netlify.app)
+- [Muhammad Saad Siddique](https://saad-muhammadsaadsiddique.vercel.app)
+- [Muhammad Ubaid Raza](https://mubaidr.js.org)[Sr. Software Engineer | Full Stack Developer | Chrome Extension Expert]
+- [Muhammad Uzair Khan](https://uzairkhan.tech)
+- [Muhammed Hisham A](https://coderhisham.com)[Full Stack Developer]
+- [Muhammet Fatih Di̇Nç](https://mfatihdinc.com)
+- [Mukul Chugh](https://mukulchugh.com)
+- [Mushfiqur Rahman](https://mushfiq.xyz)
+- [Mustapha El Alamy](https://alamy.vercel.app)[Fullstack Software Engineer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://alamy.vercel.app)